pkg/expr/transform.go

```diff
@@ -11,11 +11,35 @@ import (
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/tsdb"
+	"github.com/prometheus/client_golang/prometheus"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+var (
+	transformQueryCounter   *prometheus.CounterVec
+	transformQueryHistogram *prometheus.HistogramVec
+)
+
+func init() {
+	transformQueryCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
The counter is not really useful here since the histograms exposes a counter as well. 

`grafana_transform_queries_duration_seconds_sum{status="failure"} 0.0026933020000000003
grafana_transform_queries_duration_seconds_count`

The reason to use histograms and counters would be if we want to have more labels for the counter.